llama-index-googledocs-qa/requirements.txt

```diff
@@ -0,0 +1,153 @@
+aiofiles==23.1.0
```

Comment:
we do not need all of these dependencies, most of them are chainlit depdendencies, you can just add chainlit as a dependency

Reply:
So should I just put : 
```
llama_index
langchain
openai
chainlit
```
?